### PR TITLE
testing/ranger: resurrect and update to 1.9.1

### PR DESCRIPTION
**Pull request by jantatje**
This APKBUILD was unmaintained, didn't work and got purged.
Issue was either in ranger or ncurses, both of which have been updated in the meantime, which has fixed the segfault problem.

### CODE REVIEW

File: testing/ranger/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Maintainer: Jan Tatje <jan@jnt.io>
+
+pkgname=ranger
+pkgver=1.9.1
+pkgrel=0
+pkgdesc="A simple, vim-like file manager"
+url="https://ranger.github.io/"
+arch="noarch"
+license="GPL"
+options="!check"
```
Comment (TBK) [inline testing/ranger/APKBUILD]: Please add a comment why check is disabled.

File: testing/ranger/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Contributor: Bartłomiej Piotrowski <nospam@bpiotrowski.pl>
+# Maintainer: Jan Tatje <jan@jnt.io>
+
+pkgname=ranger
+pkgver=1.9.1
+pkgrel=0
+pkgdesc="A simple, vim-like file manager"
+url="https://ranger.github.io/"
+arch="noarch"
+license="GPL"
```
Comment (TBK) [inline testing/ranger/APKBUILD]: license="GPL-3.0" - https://github.com/ranger/ranger/blob/master/LICENSE

File: testing/ranger/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+subpackages="$pkgname-doc"
+source="https://ranger.github.io/${pkgname}-${pkgver}.tar.gz"
+
+build() {
+    return 0
+}
+
+package() { 
+    cd "$srcdir"/$pkgname-$pkgver
+    python setup.py -q install --root="${pkgdir}" --optimize=1 || return 1
```
Comment (TBK) [inline testing/ranger/APKBUILD]: " || return 1" can be removed.

File: testing/ranger/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+pkgname=ranger
+pkgver=1.9.1
+pkgrel=0
+pkgdesc="A simple, vim-like file manager"
+url="https://ranger.github.io/"
+arch="noarch"
+license="GPL"
+options="!check"
+depends="python"
```
Comment (TBK) [inline testing/ranger/APKBUILD]: Could it use python3 instead or do you need it to use python2?

**Comment by jantatje** [comment]
I have implemented the changes proposed by @TBK

**Pull request closed without merging**